src/node_file.cc

```diff
@@ -863,6 +868,26 @@ static void MKDir(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+static void RealPath(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  if (args.Length() < 1)
+    return TYPE_ERROR("path is required");
+  if (!args[0]->IsString())
+    return TYPE_ERROR("path must be a string");
+
+  node::Utf8Value path(env->isolate(), args[0]);
```

Comment:
@jasnell This should probably accept the same set of objects as the other functions you updated, right?